### PR TITLE
delet page html

### PR DESCRIPTION
**Pull request by VenkatS1208**

### CODE REVIEW

File: topbar.css [hunk 1]
```diff
@@ -37,4 +37,7 @@ body {
 
 #topbar li a:hover {
     background-color: #777777;
+}
+.w-40{
+    width: 40% !important;
```
Comment (Samyak2) [inline topbar.css]: What does this do?

**Comment by Samyak2** [comment]
Someone else has already sent a PR for the delete page. I like your design too, so can you rename the file to deletepage2.html please?
Also the mine button has been added by someone else. Copy the mine button from the master branch of my repo, so that your PR doesn't give conflicts.

File: topbar.css (hunk 1 above)
Comment (VenkatS1208) [reply topbar.css]: for custom button width, I used this one

**Comment by Samyak2** [comment]
Please replace the content of deletepage.html with the one in master branch of my repo. So that I can keep both versions.

**Pull request closed without merging**